components/resources/brave_components_strings.grd

```diff
@@ -562,6 +562,8 @@
       <message name="IDS_BRAVE_SYNC_ERROR_MISSING_SYNC_CODE_TITLE" desc="Error message in dialog for when device name is missing (description)">Please add a sync code.</message>
       <message name="IDS_BRAVE_SYNC_ERROR_INIT_FAILED_TITLE" desc="Error message in dialog for when a Sync error happens in the back-end">Couldn't contact Sync servers</message>
       <message name="IDS_BRAVE_SYNC_ERROR_INIT_FAILED_DESCRIPTION" desc="Error message in dialog for when a Sync error happens in the back-end">You might be having network trouble, or there could be a problem with the Sync servers.</message>
+      <message name="IDS_BRAVE_SYNC_REQUIRES_CORRECT_TIME_TITLE" desc="Error message in dialog for when a Sync could not couldn't connect to Sync servers, because of wrong system time (title)">Couldn't connect to Sync servers</message>
+      <message name="IDS_BRAVE_SYNC_REQUIRES_CORRECT_TIME_DESCRIPTION" desc="Error message in dialog for when a Sync could not couldn't connect to Sync servers, because of wrong system time (description)">You might be having wrong system time or timezone setup on your device. Please setup the correct time and timezone</message>
```

Comment:
Suggested text:
> "Your computer's clock may be set to the wrong time, or to the wrong time-zone. Check your clock settings.

Reply:
Followed up with suggested modal design:

![Errors](https://user-images.githubusercontent.com/2388823/55591908-b675fd80-56eb-11e9-8462-4a69b4625626.png)

Reply:
@rossmoody for error message (modal component)

Reply:
should prob go with a default button for this case. `Got it` isn't destructive action

Reply:
@karenkliu please put the design in a different issue, thanks!

Reply:
design issue created https://github.com/brave/brave-browser/issues/4013